src/test/java/com/google/api/generator/engine/format/LineCommentTest.java

```diff
@@ -0,0 +1,37 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.api.generator.engine.format;
+import static com.google.common.truth.Truth.assertThat;
+import static org.junit.Assert.assertThrows;
+
+import org.junit.Test;
+
+public class LineCommentTest {
+  @Test
+  public void writeNormalLineComment() {
+      String content = "this is a test comment";
+      LineComment lineComment = LineComment.builder().setComment(content).build();
+      String formattedComment = "// this is a test comment\n";
+      assertThat(lineComment.write()).isEqualTo(formattedComment);
+  }
+
+  @Test
+  public void writeLongLineComment() {
+    String content = "this is a long test comment with so many words, hello world, hello again, hello for 3 times, blah, blah!";
```

Comment:
Nit: If these go past the 100 char limit, didn't the linter catch this?

Reply:
`bazel run :google_java_format` seems to have passed this.